config/main.py

```diff
@@ -1188,7 +1188,8 @@ def print_dry_run_message(dry_run):
 @click.argument('patch-file-path', type=str, required=True)
 @click.option('-f', '--format', type=click.Choice([e.name for e in ConfigFormat]),
                default=ConfigFormat.CONFIGDB.name,
-               help='format of config of the patch is either ConfigDb(ABNF) or SonicYang')
+               help='format of config of the patch is either ConfigDb(ABNF) or SonicYang',
```

Comment:
>SonicYang [](http://example.com/codeflow?start=79&length=9)

Regarding the format string, we prefer ConfigYang or config_yang (ref: [https://github.com/Azure/sonic-utilities/blob/055ed4fafeaece1f00e4dcb20631805a5e621fb6/config/main.py#L1335](https://github.com/Azure/sonic-utilities/blob/055ed4fafeaece1f00e4dcb20631805a5e621fb6/config/main.py#L1335)).

The reason is to align with the naming of config_db. And in future we may have state_db/state_yang (just a conceptual example).

If you agree and would like to use another PR, we can merge this PR first.

Reply:
let me address this in a different PR